litestar/_openapi/parameters.py

```diff
@@ -160,28 +134,18 @@ def get_recursive_handler_parameters(
     """
 
     if field_name not in dependency_providers:
-        return [
-            create_parameter(
-                generate_examples=generate_examples,
-                parameter_name=field_name,
-                path_parameters=path_parameters,
-                schemas=schemas,
-                signature_field=signature_field,
-            )
-        ]
+        return [create_parameter(signature_field, field_name, path_parameters, schema_creator)]
```

Comment:
So, please use kwargs rather than positional arguments. The reason for this is two fold:

1. Kwargs are more resilient to refactoring.
2. Kwargs are more readable. 

The rule of thumb is - above two arguments always use kwargs. Below is optional but i personally prefer using them.

Reply:
Done.

FWIW I opted for positional arguments because (a) they are all required and (b) the passed parameters are (usually) variables named identically to the formal parameters so the meaning is obvious.

> Kwargs are more resilient to refactoring.

Not so sure about that; in my experience swapping the order of required parameters (which breaks positional args) is not more common than renaming parameters (which breaks kwargs).

> Kwargs are more readable.

This might be true in general, unless the caller names the passed parameters identically to the formal parameters. To me `return f(x, y, z, w)` is no less readable than `return f(x=x, y=y, z=z, w=w)`, and actually more readable than `return f(z=z, x=x, w=w, y=y)` which tends to happen over time once we stop caring about parameter order.